app/include/u8g2_fonts.h

```diff
@@ -0,0 +1,18 @@
+
+#ifndef _U8G2_FONTS_H
+#define _U8G2_FONTS_H
+
+#define U8G2_FONT_TABLE_ENTRY(font)
+
+// ***************************************************************************
+// Configure U8glib fonts
+//
+// Add a U8G2_FONT_TABLE_ENTRY for each font you want to compile into the image
+#define U8G2_FONT_TABLE \
+  U8G2_FONT_TABLE_ENTRY(font_6x10_tf) \
```

Comment:
Could you add something like the below. This allows you to define the extra fonts that you want and include them in your config that is included everywhere (or via -include on the command line). This means that this file then doesn't need to be modified to choose fonts.

```
#ifndef U8G_FONT_TABLE_EXTRA
#define U8G_FONT_TABLE_EXTRA
#endif
 #define U8G_FONT_TABLE_ENTRY(font)
 #define U8G_FONT_TABLE                          \
     U8G_FONT_TABLE_ENTRY(font_6x10)             \
    U8G_FONT_TABLE_EXTRA       \
     U8G_FONT_TABLE_ENTRY(font_chikita)
```

Reply:
Good idea, seams to work.